src/pages/cart/cart.jsx

```diff
@@ -0,0 +1,12 @@
+import { Footer } from "../../components/footer/footer";
+import { Navbar } from "../../components/navbar/navbar";
+
```

Comment:
try to make index.js in seperate folder, and import from that

Reply:
ok